src/components/Message/MessageSimple.js

```diff
@@ -206,6 +207,15 @@ const MessageSimple = (props) => {
                 reactionSelectorRef={reactionSelectorRef}
               />
             )}
+
+            {message.mml && (
```

Comment:
not sure it matters but some optional chained and some not

Reply:
yea just to suppress TS errors, types are not accurate